back/engines/free/email_campaigns/spec/factories/campaigns.rb

```diff
@@ -18,6 +18,24 @@
     end
   end
 
+  factory :manual_project_participants_campaign, class: EmailCampaigns::Campaigns::ManualProjectParticipants do
+    enabled { true }
+    author
+    project { create(:project) } # rubocop:disable FactoryBot/FactoryAssociationWithStrategy
```

Comment:
I haven't been able to get this factory to work (basic test in model spec always fails with `Project can't be blank`) by using any of the 'good' approaches listed by Rubocop docs [here](https://www.rubydoc.info/gems/rubocop-factory_bot/RuboCop/Cop/FactoryBot/FactoryAssociationWithStrategy).